cmd2/cmd2.py

```diff
@@ -38,6 +38,7 @@
 import re
 import sys
 import threading
+from code import InteractiveConsole
```

Comment:
Lots of new code is added here, but no new unit tests.  Should a few unit tests be added?

Reply:
I really just moved existing code into functions. No unit test or coverage changes.